internal/tools/wallet.go

```diff
@@ -32,7 +32,8 @@ import (
 )
 
 type FlagsWallet struct {
-	Port uint `default:"8701" flag:"port" info:"Dev wallet port to listen on"`
+	Port uint   `default:"8701" flag:"port" info:"Dev wallet port to listen on"`
+	Host string `default:"http://localhost:8080" flag:"emulator-host" info:"Host for access node connection"`
```

Comment:
```suggestion
	Host string `default:"http://localhost:8888" flag:"emulator-host" info:"Host for access node connection"`
```

It looks like the emulator bundled with the CLI no longer exposes a server on port 8080. Should this be `http://localhost:8888` now?